### PR TITLE
chore(docs): document explain REST API parameter

### PR DESCRIPTION
**Pull request by puzpuzpuz**
Core PR: https://github.com/questdb/questdb/pull/1836

### CODE REVIEW

File: docs/reference/api/rest.md [hunk 1]
```diff
@@ -310,7 +310,8 @@ closed.
...
 | `nm`      | No       | `false` | `true` or `false`. Skips the metadata section of the response when set to `true`.                                                                                                      |
 | `query`   | Yes      |         | URL encoded query text. It can be multi-line.                                                                                                                                          |
-| `timings` | No       | `false` | `true` or `false`. When set to `true`, QuestDB will also include a `timings` property in the response which gives details about the execution.                                         |
+| `timings` | No       | `false` | `true` or `false`. When set to `true`, QuestDB will also include a `timings` property in the response which gives details about the execution times.                                         |
+| `explain` | No       | `false` | `true` or `false`. When set to `true`, QuestDB will also include a `explain` property in the response which gives details about the execution decisions.                                         |
```
Comment (bsmth) [inline docs/reference/api/rest.md:314]: Thanks. I feel like we should remove "decisions" as it's a bit ambiguous what that means. How about

> details about the query execution.

Comment (puzpuzpuz) [reply docs/reference/api/rest.md]: Changed to `gives details about the execution plan` in 633a04d04f87977d170d389f5be63156f632377c. Also fixed a misprint in the same file.

File: docs/reference/api/rest.md [hunk 2]
```diff
@@ -310,7 +310,8 @@ closed.
...
 | `nm`      | No       | `false` | `true` or `false`. Skips the metadata section of the response when set to `true`.                                                                                                      |
 | `query`   | Yes      |         | URL encoded query text. It can be multi-line.                                                                                                                                          |
-| `timings` | No       | `false` | `true` or `false`. When set to `true`, QuestDB will also include a `timings` property in the response which gives details about the execution.                                         |
+| `timings` | No       | `false` | `true` or `false`. When set to `true`, QuestDB will also include a `timings` property in the response which gives details about the execution times.                                         |
+| `explain` | No       | `false` | `true` or `false`. When set to `true`, QuestDB will also include a `explain` property in the response which gives details about the execution plan.                                         |
```
Comment (bsmth) [inline docs/reference/api/rest.md:314]: typo 

>an `explain` property

**Review by bsmth — APPROVED** [review]
Minor typo outstanding, otherwise looks good, thanks for the additions!

File: docs/reference/api/rest.md (hunk 2 above)
Comment (puzpuzpuz) [reply docs/reference/api/rest.md]: > Feel free to merge after fixing

I don't have the permissions, so it's better for you to merge this one.

Comment (bsmth) [reply docs/reference/api/rest.md]: OK, will do

**Pull request merged**